cetmix_tower_yaml/tests/test_server_template.py

```diff
@@ -0,0 +1,16 @@
+from odoo.tests import TransactionCase
+
+
+class TestTowerServerTemplate(TransactionCase):
+    def setUp(self, *args, **kwargs):
+        super().setUp(*args, **kwargs)
+
+        self.ServerTemplate = self.env["cx.tower.server.template"]
+
+    def test_server_template_ssh_password_mask(self):
+        """Test that SSH password is masked in YAML"""
+        server_template = self.ServerTemplate.create(
+            {"name": "Test Server Template", "ssh_password": "test_password"}
+        )
+        record_dict = server_template._prepare_record_for_yaml()
+        self.assertEqual(record_dict["ssh_password"], "********")
```

Comment:
Add more context to the `assertEqual` message so that, in case of failure, it's clear what value was returned instead of the expected one.

Reply:
Good notice! FYI: f-strings are preferred over the .format because in most cases they are faster.